src/OpalCompiler-Core/Behavior.extension.st

```diff
@@ -70,6 +68,14 @@ Behavior >> evaluate: aString [
 	^self compiler evaluate: aString
 ]
 
+{ #category : #'*OpalCompiler-Core' }
+Behavior >> lookupLocalVar: aName [
+	^ self classLayout 
+		resolveSlot: aName asSymbol 
+		ifFound: [:var | var] 
+		ifNone: [ self innerBindingOf: aName ]
```

Comment:
The local lookup is introduced to find any variable defined in the class without the full environment lookup. It is required to allow the override of an #environment of a class during a compilation. 

We should really review and unify all current lookup methods: #lookupVar:#lookupLocalVar:/#bindingOf:/#innerBindingOf:.